src/module-1/styles/scss/components/_product.scss

```diff
@@ -0,0 +1,142 @@
+.product {
+  align-items: center;
+  background-color: white;
+  border-radius: 8px;
+  box-shadow: 0 2px 8px #00000022;
+  display: flex;
+  flex-direction: column;
+  min-height: 486px;
+  width: 400px;
+  margin: 30px;
+
+  &__img {
+    height: 272px;
+    margin-top: 31px;
+    max-width: 100%;
+    padding: 15px;
+  }
+
+  &__price {
+    align-self: flex-end;
+    margin-right: 20.49px;
```

Comment:
margin: 36px 20px auto 248px;
Use a single style for margins, paddings, widths, heights borders etc.  
Use even px values, that's a good practice in css unless you need odds for sentering or other does not fit design totally.  
Round px values to integer numbers (otherwise rounding is performed in browser anyway)